app.js

```diff
@@ -73,15 +74,17 @@ IMPORTANT DETAIL: You may not use the arithmetic operator + in this function. To
 Test this function by hand in the console to get it working, and when you think it is finished, uncomment the call for the testSumArray() function and see if the test passes.*/
 
 // Write your code here
-var testArray = [2, 3, 4]; //eslint-disable-line
+ //eslint-disable-line
+var testArray = [2, 3, 4];
 
 function sumArray(sumArr) { //eslint-disable-line
-
+  var outputArray = sum(sum(sumArr[0],sumArr[1])[0],sumArr[2])[0];
```

Comment:
I'd name this something that doesn't have the word "array" in it, since it's actually just a number... maybe `outputSum` or `outputNumber`?